cypress/integration/account_settings/sidebar/channel_switcher_spec.js

```diff
@@ -164,5 +164,22 @@ describe('Account Settings > Sidebar > Channel Switcher', () => {
 
         // * Channel switcher should still be accessible
         cy.get('#quickSwitchHint').should('be.visible');
+
```

Comment:
Added to ensure that the test key matches with the release testing steps. After looking at the corresponding ticket [MM-13216](https://mattermost.atlassian.net/browse/MM-13216), I realized that it's missing the last steps and assertion of:
- (7) Type the first few letters of a channel you want to switch to
- (8) Locate the channel using up/down arrows+enter or mouse click

Then expectation of:
- After 8) you should be redirected to the channel you selected